bcrs-ui/src/views/StateExample.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="stateExample">
+    <h1>This is an about page that contains an example for state</h1>
+    <ul>
+      <li>{{ stateText }}</li>
+    </ul>
+  </div>
+</template>
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { State } from 'vuex-class'
+
+@Component
```

Comment:
Shouldn't this be `@Component({})` to avoid a lint error?

Reply:
Currently there is no lint error without in this framework or in coops-ui. Should there be?

Reply:
Don't change this for now.